deps/rabbit/src/rabbit_stream_coordinator.erl

```diff
@@ -2092,3 +2095,16 @@ transfer_leadership([Destination | _] = _TransferCandidates) ->
         undefined ->
             {ok, undefined}
     end.
+
+online_coordinator_members() ->
+    lists:filter(fun erlang:is_pid/1,
+                 rabbit_misc:append_rpc_all_nodes(rabbit_nodes:all_running(),
+                                                  erlang, whereis, [rabbit_stream_coordinator])).
+
+is_quorum_critical() ->
+    NumAlive = length(online_coordinator_members()),
+    case ra:members(rabbit_stream_coordinator) of
```

Comment:
Does this mean that if the local stream coordinator member isn't running we say it is ok to go ahead?